lib/hexo.js

```diff
@@ -34,8 +34,9 @@ function entry(cwd = process.cwd(), args) {
 
     hexo.base_dir = path;
 
-    return loadModule(path, args).catch(() => {
-      log.error('Local hexo not found in %s', chalk.magenta(tildify(path)));
+    return loadModule(path, args).catch(err => {
+      log.error(err.message);
+      log.error('Local hexo loading failed in %s', chalk.magenta(tildify(path)));
       log.error('Try running: \'npm install hexo --save\'');
```

Comment:
```suggestion
      log.error('Try running: \'rm -rf node_modules && npm install\'');
```

Firstly, `--save` is no longer required for `npm install --force`.
Secondly,  in most cases the `node_modules` has been corrupted. The better idea is to let user remove `node_modules` and run `npm i` again (with [npm cache disabled](https://docs.npmjs.com/misc/config#force))